### PR TITLE
Add Context Menu Option to Run Application

### PR DESCRIPTION
**Pull request by patrick-hovsepian**
image

Need to add locales but opening up in case there's wording change requests

### CODE REVIEW

File: src/Ryujinx.Ava/Assets/Locales/en_US.json [hunk 1]
```diff
@@ -282,6 +282,7 @@
   "ControllerSettingsSaveProfileToolTip": "Save Profile",
   "MenuBarFileToolsTakeScreenshot": "Take Screenshot",
   "MenuBarFileToolsHideUi": "Hide UI",
+  "GameListContextMenuRunApplication": "Run Application",
```
Comment (yell0wsuit) [inline src/Ryujinx.Ava/Assets/Locales/en_US.json:292]: You should add newly made string at the bottom to make it easier to localize

Comment (patrick-hovsepian) [reply src/Ryujinx.Ava/Assets/Locales/en_US.json:292]: it's right above the prop for the toggle favorites which is it's neighbor item in the menu so i figured that was a better logical grouping. I'd add the others in the same place

Comment (AcK77) [reply src/Ryujinx.Ava/Assets/Locales/en_US.json:292]: Order doesn’t matter since translations are done using crowdin

**Review by AcK77 — APPROVED** [review]
Lgtm

File: src/Ryujinx.Ava/Assets/Locales/en_US.json (hunk 1 above)
Comment (yell0wsuit) [reply src/Ryujinx.Ava/Assets/Locales/en_US.json:292]: Oh, didn't know about that. I thought adding at the bottom would make it easier to compare diff.

**Pull request merged**